WiFi_Sit_N__Wheelie_receiver/WiFi_Sit_N__Wheelie_receiver.ino

```diff
@@ -0,0 +1,104 @@
+#include <SPI.h>
+#include <Adafruit_WINC1500.h>
+#include <Adafruit_WINC1500Udp.h>
+
+//Define w1 and w2 as analog output to control the BLDC
+const int w1 = 6;
+const int w2 = 14;
+
+// Define the WINC1500 board connections below.
+// If you're following the Adafruit WINC1500 board
+// guide you don't need to modify these:
+#define WINC_CS   31
+#define WINC_IRQ  23
+#define WINC_RST  7
+#define WINC_EN   5     // or, tie EN to VCC and comment this out
+// The SPI pins of the WINC1500 (SCK, MOSI, MISO) should be
+// connected to the hardware SPI port of the Arduino.
+
+// Setup the WINC1500 connection with the pins above and the default hardware SPI.
+Adafruit_WINC1500 WiFi(WINC_CS, WINC_IRQ, WINC_RST);
+
+//Adafruit_WINC1500 WiFi;
+int status = WL_IDLE_STATUS;
+char ssid[] = "SSID";  //  your network SSID (name)
+char pass[] = "Password!";       // your network password
+int keyIndex = 0;            // your network key Index number (needed only for WEP)
```

Comment:
this 0 should be defined above and then used here following the example below. This is a good practice so that we don't just have magic numbers. 
```suggestion
//defined with other defines
#define NETWORK_KEY_INDEX 0

//down with other code
int keyIndex = NETWORK_KEY_INDEX;            // your network key Index number (needed only for WEP)
```

Reply:
comment here to record the conversation

Reply:
does you key index change?

Reply:
yes I know it's in the example of the library, but the way you are supposed to do it is define it above then you only use the variable name in code that way you only have to change it one time rather than finding all the magic numbers in code